packages/desktop-gui/src/update/updates.js

```diff
@@ -5,16 +5,23 @@ import updateStore from '../update/update-store'
 import ipc from '../lib/ipc'
 import { useLifecycle } from '../lib/use-lifecycle'
 
+let initialLaunch = true
```

Comment:
hm, i had assumed that `initialLaunch` would be like... "is this the first time Cypress has been opened on this machine ID?", but it appears to be measuring "is this the first update check of a new Cypress application session?". is my understanding correct?

Reply:
correct. application session is desired behavior